== TEST (2019-05-23 18:30:52) 
-- CODE:
from numpy import*
n=ones(6,dtype=int)
print(n)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-23 18:31:00) 
-- CODE:
from numpy import*
n=ones(6,dtype=int)
print(n)
-- EXECUTION TIME:
0.368873
-- TEST CASE 1:
---- input:
34
---- correct output:
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
---- user output:
[1 1 1 1 1 1]
-- TEST CASE 2:
---- input:
10
---- correct output:
[1 1 1 1 1 1 1 1 1 1]
---- user output:
[1 1 1 1 1 1]
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-23 18:31:08) 
-- CODE:
from numpy import*
n=ones(,dtype=int)
print(n)
-- EXECUTION TIME:
0.225953
-- ERROR:
  File "/home/codebench/codes/1771/672/main.py", line 2
    n=ones(,dtype=int)
           ^
SyntaxError: invalid syntax

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-23 18:31:11) 
-- CODE:
from numpy import*
n=ones(,dtype=int)
print(n)
-- ERROR:
File "XXXX", line 2
    n=ones(,dtype=int)
           ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-23 18:32:32) 
-- CODE:
from numpy import*
n=int(input("digite um numero: ")_
n1=ones(n)

print(n1)
-- ERROR:
File "XXXX", line 2
    n=int(input()_
                 ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-23 18:32:42) 
-- CODE:
from numpy import*
n=int(input("digite um numero: "))
n1=ones(n)

print(n1)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-23 18:32:58) 
-- CODE:
from numpy import*
n=int(input("digite um numero: "))
n1=ones(n,dtype=int)

print(n1)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-23 18:33:02) 
-- CODE:
from numpy import*
n=int(input("digite um numero: "))
n1=ones(n,dtype=int)

print(n1)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-23 18:33:04) 
-- CODE:
from numpy import*
n=int(input("digite um numero: "))
n1=ones(n,dtype=int)

print(n1)
-- EXECUTION TIME:
0.497831
-- TEST CASE 1:
---- input:
34
---- correct output:
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
---- user output:
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
-- TEST CASE 2:
---- input:
10
---- correct output:
[1 1 1 1 1 1 1 1 1 1]
---- user output:
[1 1 1 1 1 1 1 1 1 1]
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
